Gemfile

```diff
@@ -4,11 +4,3 @@ source "https://rubygems.org"
 ruby "2.4.1"
 
 gemspec path: "."
-gemspec path: "decidim-core"
-gemspec path: "decidim-system"
-gemspec path: "decidim-admin"
-gemspec path: "decidim-dev"
-gemspec path: "decidim-api"
-gemspec path: "decidim-pages"
-gemspec path: "decidim-comments"
-gemspec path: "decidim-meetings"
```

Comment:
Super nice catch! Trying to fix all that. Thanks for the help you've been giving us! 👍